src/App.js

```diff
@@ -1,14 +1,23 @@
 import "./App.css";
+// import MockAPI from './mockman/MockAPI'
+import { Routes, Route } from "react-router-dom";
 import "./components/navbar.css"
 import LandingPage from "./page/landingPage";
 import SingleVideoPage from "./page/SingleVideoPage";
 import VideoListing from "./page/videoListing";
+import WatchLaterpage from "./page/WatchLaterpage";
+import { HistoryPage } from "./page/History-page";
```

Comment:
Don't mix named or default import/import in your projects . 
Either use named or default export/import.